src/climate_assessment/climate/ciceroscm.py

```diff
@@ -19,7 +19,7 @@ def get_ciceroscm_configurations(
         # version strings for linux and windows might be different!
         raise AssertionError(CICEROSCM.get_version())
 
-    with open(ciceroscm_probabilistic_file, "r") as fh:
+    with open(ciceroscm_probabilistic_file) as fh:
```

Comment:
Ah yes just checked that r is the default. I'd personally prefer keeping it because it makes it easier to understand what's happening (reading vs writing) but if that's the ruff style fine with me.

Reply:
Ruff style, fighting it not worth it so going to leave it